MultimodalQnA/tests/test_compose_on_gaudi.sh

```diff
@@ -201,7 +201,24 @@ function validate_microservices() {
         "retriever-multimodal-redis" \
         "{\"text\":\"test\",\"embedding\":${your_embedding}}"
 
-    sleep 3m
+    echo "Wait for lvm-llava service to be ready"
+    max_retries=10
+    for i in $(seq $max_retries)
+    do
+        lvm_logs=$(docker logs lvm-llava 2>&1 | grep "Uvicorn running on http://" || true)
```

Comment:
With `set -e`, this line was causing the script to exit because `grep` returns a non-zero exit status when no match is found. The addition of `|| true` gets around this and makes the script behave as intended.